### ISSUE TITLE
Open ports with open-input-file are not closed

### ISSUE THREAD

**Issue by arthurmaciel** [cyclone-scheme/winds#27]

**Comment by arthurmaciel** [comment]
@justinethier, just for safety, could you also please test? 

#34 fixes this bug

**Comment by justinethier** [comment]
@arthurmaciel The changes themselves look good, let me get them pulled in and tested.

Just off the bat, I do notice there are several other cases of ports being opened but not closed. Do we want to patch these lines as well?

```
sys/update-packages-wiki.scm:51:          (display wiki-new-content (open-output-file wiki-file-path))
sys/update-wiki-index.scm:41:           (open-output-file *default-wiki-home-file*)))
sys/update-library-index.scm:22:                (open-output-file *default-library-index*)))

sys/update-wiki-index.scm|18| (pkg (metadata->pkg (cdr (read (open-input-file md-path))))))
sys/update-library-index.scm|7| (let ((content (read (open-input-file sld))))    
sys/tasks.scm|21| (define file-port (open-input-file file))                      
winds.scm|30| (open-input-file (->path work-dir                                  
winds.scm|56| (open-input-file (->path work-dir                                  
winds.scm|68| (let ((md (cdr (read (open-input-file metadata-path)))))   
```

**Comment by arthurmaciel** [comment]
Thanks for the heads-up, Justin!

**Comment by arthurmaciel** [comment]
Fixed with #35.
